Find a functionally equivalent code:android.media.MediaRouter.UserRouteInfo.setVolumeHandling:COMMENT
Method Modifier: public      
Comment:/**
 * Defines whether volume for the playback associated with this route is fixed
 * ({@link RouteInfo#PLAYBACK_VOLUME_FIXED}) or can modified
 * ({@link RouteInfo#PLAYBACK_VOLUME_VARIABLE}).
 * @param volumeHandling
 */

@@ -1,6 +1,6 @@
 {
     if (mVolumeHandling != volumeHandling) {
         mVolumeHandling = volumeHandling;
-        setPlaybackInfoOnRcc(RemoteControlClient.PLAYBACKINFO_VOLUME_HANDLING, volumeHandling);
+        configureSessionVolume();
     }
 }

